kustomization.yaml

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
In many projects, it's custom to put these files in `deploy/` or even `deploy/kubernetes`, though given this project is Kubernetes-only that may be superfluous.

Furthermore, from a `kustomize` point of view, this is the `base` layer, so `deploy/base` or `deploy/kubernetes/base` seems appropriate. This allows to ship some so-called _overlays_ (e.g., to deploy `canary` images) as well.

Reply:
I have updated the code to move the manifests to `deploy/base`